source/adios2/toolkit/transport/file/FilePointer.cpp

```diff
@@ -48,16 +48,16 @@ void FilePointer::Open(const std::string &name, const OpenMode openMode)
 
     if (m_OpenMode == OpenMode::Write)
     {
-        m_File = std::fopen(name.c_str(), "w");
+        m_File = fopen(name.c_str(), "wb");
```

Comment:
the _s functions are a windows thing I believe.  Other OSs have thier own versions (OpenBSD has a different version).  You should be able to remove the warning though with:
```cpp
#ifdef _MSC_VER
#define _CRT_SECURE_NO_WARNINGS
#endif
```
at the top.  I do think it would still be useful to keep the C stdio implementation since the C stdio operations tends to be much more performance than the C++ iostream.  If instead, though, you don't want to bother with it and just delete the backend completely that's fine too, but since it's already there I think it's useful to keep around.